src/plugins/add-cal/add-calendar.js

```diff
@@ -46,12 +46,12 @@ var componentName = "wb-addcal",
 					en: {
 						"addcal-addto": "Add to",
 						"addcal-calendar": "calendar",
-						"addcal-ical": "iCal format (iPhone, Outlook...)"
+						"addcal-ical": "Other (Outlook, Apple, etc.)"
 					},
 					fr: {
 						"addcal-addto": "Ajouter au",
 						"addcal-calendar": "calendrier",
-						"addcal-ical": "Format iCal (iPhone, Outlook....)"
+						"addcal-ical": "Autre (Outlook, Apple, etc.)"
 					}
```

Comment:
For stabilization, those i18n text will need to be moved into our i18n csv. But for a transition we can keep both where the i18n csv override the internal plugin value.